src/svgToJsx.js

```diff
@@ -1,26 +1,25 @@
+const { parse } = require("babylon");
+const traverse = require("babel-traverse").default;
+const { optimize } = require("svgo");
 
-const {parse} = require('babylon');
-const traverse = require('babel-traverse').default;
-const SVGO = require('svgo');
-var svg = new SVGO({plugins: [
-  {removeViewBox: false},
-]});
-
-const isSvgElement = require('./isSvgElement');
-const addSpreadAttribute = require('./addSpreadAttribute');
-const camelizeAttribute = require('./camelizeAttribute');
-const parameterizeColors = require('./parameterizeColors');
+const isSvgElement = require("./isSvgElement");
+const addSpreadAttribute = require("./addSpreadAttribute");
+const camelizeAttribute = require("./camelizeAttribute");
+const parameterizeColors = require("./parameterizeColors");
 
 module.exports = function svgToJsx(svgString) {
-  return svg.optimize(svgString).then(r => convert(r.data));
+  return new Promise((resolve) => {
+    const result = optimize(svgString);
+    resolve(convert(result.data));
+  });
 };
 
 function convert(jsxLikeSvg) {
-  return transform(parse(jsxLikeSvg, {plugins: ['jsx']}));
+  return transform(parse(jsxLikeSvg, { plugins: ["jsx"] }));
```

Comment:
can you use the same prettier config that we use in the CMS? Will result in less changes

Reply:
It's done!